README.md

```diff
@@ -110,7 +110,7 @@ Options:
   -p, --processes INTEGER         Number of processes to be used to create the
                                   dataset. Default = 1.
 
-  -z, --gzipped                   If True, this flag indicates that the
+  -z, --is_pdb_gzipped            If True, this flag indicates that the
```

Comment:
Be careful about making this kind of change once we've published as it would be a breaking change for anyone using this flag in a script. Totally fine to do this at the moment though, as we're < 1.0.0.

Reply:
This was to avoid confusion with ` --compression_gzip` but yeah I agree